tests/ops/functions/test_matrix.py

```diff
@@ -165,6 +165,11 @@ def test_multiple_operations_tape(self):
         expected_matrix = I_CNOT @ X_S_H
         assert np.allclose(matrix, expected_matrix)
 
+        qs = qml.tape.QuantumScript(tape.operations)
+        qs_matrix = qml.matrix(tape, wire_order)
+
+        assert np.allclose(qs_matrix, expected_matrix)
+

```

Comment:
Instead of computing the same matrix twice, why not replace the `QuantumTape` with `QuantumScript`?

Reply:
Same thing for all tests.

Reply:
For the time being, I want to be overcautious with tests and run the tests for both `QuantumTape` and `QuantumScript`.  While they *shouldn't* give different results, testing is about making sure *shouldn't* actually holds true.